Source/RapyutaSimulationPlugins/Private/Tools/SimulationState.cpp

```diff
@@ -241,48 +245,55 @@ void ASimulationState::SpawnEntitySrv(UROS2GenericSrv* Service)
 {

```

Comment:
Is it possible to integrate witth URRUObjectUtils::SpawnSimActor?(not necessary but curious)

Reply:
Probably yes, just there is one point I am still not clear about the use of `SpawnActorDeferred` here, which makes this spawning a bit too specific, thus not as a straight spawn as `URRUObjectUtils::SpawnSimActor()`.